index.html

```diff
@@ -54,6 +53,8 @@
 	<script src="modules/organisation/organisation-controller.js"></script>
 	<script src="modules/organisation/add-organisation-controller.js"></script>
 	<script src="modules/lead-meeting/lead-meeting-controller.js"></script>
+	
+	<title>Ro i salen {{ organization_name }}</title>
```

Comment:
Mest en OCD-aktig greie. Flytting av taggen sørger for at alle skriptene er lastet før den definerer organization_name.